index.html

```diff
@@ -13,7 +13,7 @@
 <body>
     <div class="container">
         <div class="card1">
```

Comment:
- [ ] Please merge this project with the header section project you should create the portfolio in one doc.